### ISSUE TITLE
Disassembled code contains cells

### ISSUE THREAD

**Issue by ingria** [ton-community/disassembler#5]
For example, disassembled `EQDSbgHX03B9_0cNBAMdlmhVbvhRNYhZNhTRH4wfNBmisKB5`:

```
SETCP0
x{}
 x{2_}
  x{2_}
   x{2_}
    x{DD7022D0332273D721D70B0022C7008E1720C0009E22821032DAB09DF140015F06DB30E05F06DB30E022D31F3421C001992077F140015F07DB30E02321F140015F07}
    x{2_}
     x{B9C7EDBF3_}
      x{70ED44D020F404323220D68032ED47226F8C236F8C216F8C20ED575F04706875A1607FBA956878A16031DEED476F11D70BFFBAF2E064F800FA403020C8C9FB048103E870810080C871CF0B0122CF0A0071CF40F828CF1624CF1623FA0271CF4070FA0270FA028040CF40F823CF0B1F72CF4020C922FB005F0530C8ED476F12}
       x{01F400ED476F13CF16ED476F11CF1620C9ED5430706ADB30}
     x{62_}
      x{B3372EAC}
       x{70ED44D020F404323220D68032ED47226F8C236F8C216F8C20ED575F0430F8008064ED476F128040F40E93D31FD19170E28065ED476F128040F40E93D3FFD19170E2C8821031372EAC821080000000B1CF0B1F22CF0B1F21CF0BFFC873CF0B01F828CF1672CF40F825CF0B3F8021CF4020CF3522CF31BC9671CF4021CF17}
        x{9571CF4121CDE220C971FB005B726ADB30}
      x{B2DAB09D70ED44D020F404323220D68032ED47226F8C236F8C216F8C20ED575F0430C8ED476F1201F400ED476F13CF16ED476F11CF1620C9ED5430706ADB30}
   x{BC82650604_}
    x{707082081B7740ED44D020F404323420810080D7458E12D33F0133D33F01322071D74594807BF2F0DEDEC82401F40023CF0B3F22CF0B3F71CF4121CF1620C9ED545F0500F800ED47C8810100CF40C9D06F8CED44D0F404D71D80596F8C016F8CED57D31FD3FF30ED47206F1130706875A1607FBA956878A16031DEC8CBFFC9}
     x{D06F51ED57ED47206F1223C8CB1F8064588040F4436F52ED57ED47206F1222C8CBFF8065588040F4436F52ED575BC8ED476F1201F400ED476F13CF16ED476F11CF1620C9ED5430706A}
  x{FF}
   x{7F21D520C70191708E1220810200D721D70BFF22F9015321F910F2A8E222D31FD33F3520707070ED44D020F404323420810080D74598D33F0133D33F01329682081B774032E2702326B926F8238103E8A825A0B9B08E17C82501F40026CF0B3F23CF0B3F22CF1620C9ED547F3230DE055F05992422F140015F0ADB30E0807C}
    x{F2F05F0A}
 32 DICTPUSHCONST
s2 PUSH
1 EQINT
<{
  DICTIGETJMP
}> PUSHCONT
IFNOTJMP
PUSHREFCONT
c3 POP
ROT
s0 POP
DICTUGETJMP
x{}
 x{A00000000730DB30}
 32 DICTPUSHCONST
DICTUGETJMP
```

**Comment by ex3ndr** [comment]
Yes, this is true. You have to know what it is and then disassemble special constructions, but I still feel that something is broken here.

**Comment by dvlkv** [comment]
Marking as resolved

**Comment by dvlkv** [comment]
Fixed in v2.0.0
